src/site/topic21.rst

```diff
@@ -34,6 +34,49 @@ Tree Definition
    :align: center
 
 
+
+Measuring
+---------
+
+* A **path** is the sequence of nodes and edges leading from one node to another
+
+* The **path length** is the number of *edges* in the path
+
+* The **height** of a tree is the length of the longest path from the root to a leaf node
```

Comment:
This implies a non-empty tree could have a height of zero, or the same height as the empty tree. I tend not to prefer this convention and treat height as the number of "levels" in the tree.